keyboards/jlw/vault35_katana/config.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#define ENCODER_MAP_ENABLE
+
+#define WS2812_DI_PIN B15
+#define RGB_MATRIX_LED_COUNT 12
+#define DRIVER_LED_TOTAL RGB_MATRIX_LED_COUNT
```

Comment:
```suggestion
#define RGB_MATRIX_LED_COUNT 12
```
Pin moved to `info.json`, rest is unnecessary